flutter/lib/resources/config_manager.dart

```diff
@@ -12,6 +12,8 @@ import 'utils.dart';
 const _configListFileName = 'benchmarksConfigurations.json';
 const _defaultConfigName = 'default';
 const _defaultConfigUrl = 'asset://assets/tasks.pbtxt';
+const _offlineConfigName = 'offline';
+const _offlineConfigUrl = 'asset://assets/tasks_offline.pbtxt';
```

Comment:
Adding a new duplicated task file with only some modified path will make it harder to maintain  it. Since the app has the ability to use custom task file, it should be preferred.

Reply:
any suggestion on how to go about it? The process of using a custom task config file is quite cumbersome.

Reply:
Yes, it's not convenient but it's one-time work to add a custom task file. We also have other issue to improve the UX of it (https://github.com/mlcommons/mobile_app_open/issues/383).

For this PR, I think just adding the option to chose other task file should be sufficient.

Reply:
To be concrete, I mean just adding a new button `Choose other config` which then show the `TaskConfigScreen` should be sufficient. No need to add the `tasks_offline.pbtxt`.